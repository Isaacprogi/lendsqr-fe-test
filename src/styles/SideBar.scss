@import '../styles/variables/mixins';


.sidebar{
    min-height: 100%;
    font-family: sans-serif;
    box-shadow: 10px 5px 20px rgba(0, 0, 0, 0.04);
    display: block;
    background-color :$whiteColor;
    width: 100%;
    padding-bottom: 1rem;
    
    
   .profile-more-options{
       display: flex;
       flex-direction: column;
       align-items: flex-start;
       justify-content: flex-start;
       font-family: 'Work Sans';
       font-style: normal;
       font-weight: 400;
       font-size: 16px;
       line-height: 19px;
       color: $textPrimary;
       gap: 1rem;
       border-bottom: 1px solid rgb(236, 234, 231);

       .profile-info{
        display: flex;
        gap: 1rem;
        padding: 1rem;
        align-items: center;
        width: 100%;
        margin-top: 2rem;

        .avatar{
            border-radius: 100%;
            overflow: hidden;
            width: 3rem;
            height: 3rem;

            img{
                object-fit: cover;
                object-position: center;
                width: 100%;
                height: 100%;
            }
        }
       }


       .org{
        margin-top: 2rem;
       }
       
       >a{
           width: 100%;
           display: flex;
           align-items: center;
           justify-content: flex-start;
           cursor: pointer;
           text-decoration: none;
           padding: 1rem;
           padding-left: 2rem;

           &:focus{
            background-color: #d9ebeb;
        }

         &:hover{
            background-color: #e3f3f3;
          }
           
           .top-bottom-icon-duo {
               display: flex;
               align-items: center;
               justify-content: center;
               flex-direction: column;
               margin-right: .4rem;

               .dash-board-bottom{
                   margin-top: -5px;
               }
           }
       }

       @include mediumDevices {
        display: none;     
    }

   }
   .top{
       display: flex;
       flex-direction: column;
       align-items: flex-start;
       justify-content: flex-start;
       font-family: 'Work Sans';
       font-style: normal;
       font-weight: 400;
       font-size: 16px;
       line-height: 19px;
       color: $textPrimary;
       gap: 1rem;


       .org{
        margin-top: 2rem;
       }
       
       >a{
           width: 100%;
           display: flex;
           align-items: center;
           justify-content: flex-start;
           cursor: pointer;
           text-decoration: none;
           padding: 1rem;
           padding-left: 2rem;

           &:focus{
            background-color: #d9ebeb;
        }

         &:hover{
            background-color: #e3f3f3;
          }
           
           .top-bottom-icon-duo {
               display: flex;
               align-items: center;
               justify-content: center;
               flex-direction: column;
               margin-right: .4rem;

               .dash-board-bottom{
                   margin-top: -5px;
               }
           }
       }

       

   }
   .middle{
       display: flex;
       flex-direction: column;
       align-items: flex-start;
       width: 100%;

       h3{
         width: 100%;
         margin-top: 2rem;
         margin-bottom: 1rem;
         padding-left: 2rem;
       }

       a{
           width: 100%;
           padding-left: 2rem;
           padding-top: 1rem ;
           padding-bottom: 1rem;
           cursor: pointer;
           text-decoration: none;
           display: flex;
           align-items: center;
          
           &:focus{
               background-color: #d9ebeb;
           }

           &:hover{
               background-color: #e3f3f3;
           }

           img{
               margin-right:.5rem;
           }        
       }

       .end-section{
        width: 100%;
        padding-bottom: 2rem;
        border-bottom: .5px solid rgb(230, 221, 221);
        display: flex;
        flex-direction: column;
        align-items: flex-start;
        
     }

     .log-out-section{
         display: flex;
         flex-direction: column;
         align-items: flex-start;
         padding-left: 2rem;
         color: $textPrimary;

         img{
             margin-right: .5rem;
         }

         .log-out{
             margin-top: 3rem;
             font-weight: 400;
             font-size: 16px;

         }
         .v1{
             margin-top: 2rem;
         }
     }
   }
   .bottom{
       background-color: black;
   }

    
}