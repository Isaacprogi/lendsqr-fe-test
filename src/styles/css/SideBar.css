.sidebar {
  min-height: 100%;
  font-family: sans-serif;
  -webkit-box-shadow: 10px 5px 20px rgba(0, 0, 0, 0.04);
          box-shadow: 10px 5px 20px rgba(0, 0, 0, 0.04);
  display: block;
  background-color: white;
  width: 100%;
  padding-bottom: 1rem;
}
.sidebar .profile-more-options {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  -webkit-box-align: start;
      -ms-flex-align: start;
          align-items: flex-start;
  -webkit-box-pack: start;
      -ms-flex-pack: start;
          justify-content: flex-start;
  font-family: "Work Sans";
  font-style: normal;
  font-weight: 400;
  font-size: 16px;
  line-height: 19px;
  color: #213F7D;
  gap: 1rem;
  border-bottom: 1px solid rgb(236, 234, 231);
}
.sidebar .profile-more-options .profile-info {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  gap: 1rem;
  padding: 1rem;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  width: 100%;
  margin-top: 2rem;
}
.sidebar .profile-more-options .profile-info .avatar {
  border-radius: 100%;
  overflow: hidden;
  width: 3rem;
  height: 3rem;
}
.sidebar .profile-more-options .profile-info .avatar img {
  -o-object-fit: cover;
     object-fit: cover;
  -o-object-position: center;
     object-position: center;
  width: 100%;
  height: 100%;
}
.sidebar .profile-more-options .org {
  margin-top: 2rem;
}
.sidebar .profile-more-options > a {
  width: 100%;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  -webkit-box-pack: start;
      -ms-flex-pack: start;
          justify-content: flex-start;
  cursor: pointer;
  text-decoration: none;
  padding: 1rem;
  padding-left: 2rem;
}
.sidebar .profile-more-options > a:focus {
  background-color: #d9ebeb;
}
.sidebar .profile-more-options > a:hover {
  background-color: #e3f3f3;
}
.sidebar .profile-more-options > a .top-bottom-icon-duo {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  margin-right: 0.4rem;
}
.sidebar .profile-more-options > a .top-bottom-icon-duo .dash-board-bottom {
  margin-top: -5px;
}
@media screen and (min-width: 768px) {
  .sidebar .profile-more-options {
    display: none;
  }
}
.sidebar .top {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  -webkit-box-align: start;
      -ms-flex-align: start;
          align-items: flex-start;
  -webkit-box-pack: start;
      -ms-flex-pack: start;
          justify-content: flex-start;
  font-family: "Work Sans";
  font-style: normal;
  font-weight: 400;
  font-size: 16px;
  line-height: 19px;
  color: #213F7D;
  gap: 1rem;
}
.sidebar .top .org {
  margin-top: 2rem;
}
.sidebar .top > a {
  width: 100%;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  -webkit-box-pack: start;
      -ms-flex-pack: start;
          justify-content: flex-start;
  cursor: pointer;
  text-decoration: none;
  padding: 1rem;
  padding-left: 2rem;
}
.sidebar .top > a:focus {
  background-color: #d9ebeb;
}
.sidebar .top > a:hover {
  background-color: #e3f3f3;
}
.sidebar .top > a .top-bottom-icon-duo {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  margin-right: 0.4rem;
}
.sidebar .top > a .top-bottom-icon-duo .dash-board-bottom {
  margin-top: -5px;
}
.sidebar .middle {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  -webkit-box-align: start;
      -ms-flex-align: start;
          align-items: flex-start;
  width: 100%;
}
.sidebar .middle h3 {
  width: 100%;
  margin-top: 2rem;
  margin-bottom: 1rem;
  padding-left: 2rem;
}
.sidebar .middle a {
  width: 100%;
  padding-left: 2rem;
  padding-top: 1rem;
  padding-bottom: 1rem;
  cursor: pointer;
  text-decoration: none;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
}
.sidebar .middle a:focus {
  background-color: #d9ebeb;
}
.sidebar .middle a:hover {
  background-color: #e3f3f3;
}
.sidebar .middle a img {
  margin-right: 0.5rem;
}
.sidebar .middle .end-section {
  width: 100%;
  padding-bottom: 2rem;
  border-bottom: 0.5px solid rgb(230, 221, 221);
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  -webkit-box-align: start;
      -ms-flex-align: start;
          align-items: flex-start;
}
.sidebar .middle .log-out-section {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  -webkit-box-align: start;
      -ms-flex-align: start;
          align-items: flex-start;
  padding-left: 2rem;
  color: #213F7D;
}
.sidebar .middle .log-out-section img {
  margin-right: 0.5rem;
}
.sidebar .middle .log-out-section .log-out {
  margin-top: 3rem;
  font-weight: 400;
  font-size: 16px;
}
.sidebar .middle .log-out-section .v1 {
  margin-top: 2rem;
}
.sidebar .bottom {
  background-color: black;
}/*# sourceMappingURL=SideBar.css.map */