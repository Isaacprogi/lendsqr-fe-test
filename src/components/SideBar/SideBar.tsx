import '../../styles/css/SideBar.css'
import DashBoardTop from '../../assets/svg/sidebar/DashBoardTop.svg'
import DashBoardBottom from '../../assets/svg/sidebar/DashBoardBottom.svg'
import OrganizationTop from '../../assets/svg/sidebar/OrganizationTop.svg'
import OrganizationBottom from '../../assets/svg/sidebar/OrganizationBottom.svg'
import Users from '../../assets/svg/sidebar/Users.svg'
import Guarantors from '../../assets/svg/sidebar/Guarantors.svg'
import Decison from '../../assets/svg/sidebar/Decision.svg'
import Fees from '../../assets/svg/sidebar/fees.svg'
import Loans from '../../assets/svg/sidebar/Loans.svg'
import Savings from '../../assets/svg/sidebar/Savings.svg'
import Transactions from '../../assets/svg/sidebar/trans.svg'
import SavingsProd from '../../assets/svg/sidebar/SavingsNew.svg'
import LoanRequest from '../../assets/svg/sidebar/LoanRequests.svg'
import WhiteList from '../../assets/svg/sidebar/WhiteList.svg'
import Karma from '../../assets/svg/sidebar/Karma.svg'
import Organization from '../../assets/svg/sidebar/briefcase.svg'
import Services from '../../assets/svg/sidebar/Services.svg'
import ServiceAmount from '../../assets/svg/sidebar/ServiceAmount.svg'
import SettleMents from '../../assets/svg/sidebar/settlements.svg'
import Reports from '../../assets/svg/sidebar/chart-bar.svg'
import Preferences from '../../assets/svg/sidebar/Preferences.svg'
import Pricing from '../../assets/svg/sidebar/pricing.svg'
import AuditLogs from '../../assets/svg/sidebar/auditLogs.svg'
import SystemMessages from '../../assets/svg/sidebar/sysmessages.svg'
import Logout from '../../assets/svg/sidebar/sign-out.svg'
import { AiFillBell } from 'react-icons/ai'
import { Link } from 'react-router-dom'
import { HiDocument } from 'react-icons/hi'
import Me from '../../assets/images/Profile.png'
import { useNavigate } from 'react-router-dom'
import { BiChevronDown } from 'react-icons/bi'





export const SideBar = ({ setShowSideBar }: { setShowSideBar?: React.Dispatch<React.SetStateAction<boolean>> | undefined }) => {
    const hanldeSetShowSideBar = () => setShowSideBar && setShowSideBar(false)
    const navigate = useNavigate()
    return (
        <div className="sidebar">
            <section className='profile-more-options' data-testid='profile-more-options'>
                <div className='profile-info'>
                    <span className='avatar'>
                        <img data-testid='avatar' src={Me} alt="" />
                    </span>
                    <span data-testid="user-name-2">Adeji</span>
                </div>

                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <div className='top-bottom-icon-duo'>
                    <AiFillBell data-testid='notification-drawer-icon' />
                    </div>
                    <span> Notifications</span>
                </Link>
                
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <div className='top-bottom-icon-duo'>
                    <HiDocument data-testid='docs-drawer-icon' />
                    </div>
                    <span>Docs</span>
                </Link>
                    
            </section>
            <section className="top">
                <Link onClick={hanldeSetShowSideBar} to='/' className='org'>
                    <div className='top-bottom-icon-duo'>
                        <img src={OrganizationTop} alt="switch-organization-icon-1" />
                        <img src={OrganizationBottom} alt="switch-organization-icon-2" />
                    </div>
                    <span>Switch Organization</span>
                    <BiChevronDown/>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <div className='top-bottom-icon-duo'>
                        <img src={DashBoardTop} alt="dashboard-icons-1" />
                        <img className='dash-board-bottom' src={DashBoardBottom} alt="dashboard-icon-2" />
                    </div>
                    <span>Dash Board</span>
                </Link>
            </section>
            <section className="middle">

                {/* top section */}

                <h3>Customers</h3>
                <Link onClick={hanldeSetShowSideBar} to='/users'>
                    <img src={Users} alt="users-icon" />
                    <span>Users</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={Guarantors} alt="guarantors-icon" />
                    <span>Guarantors</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={Loans} alt="loans-icon" />
                    <span>Loans</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={Decison} alt="decision-icon" />
                    <span>Decision Model</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={Savings} alt="savings-icon" />
                    <span>Savings</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={LoanRequest} alt="loan-request-icon" />
                    <span>Loan Requests</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={WhiteList} alt="white-list-icon" />
                    <span>whiteList</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={Karma} alt="karma-icon" />
                    <span>Karma</span>
                </Link>

                {/* middle section */}


                <h3>Businesses</h3>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={Organization} alt="organization-icon" />
                    <span>Organization</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={Guarantors} alt="loan-products-icon" />
                    <span>Loan Products</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={SavingsProd} alt="savings-product-icon" />
                    <span>Savings Products</span>
                </Link>
                <Link to='/'>
                    <img src={Fees} alt="fees-and-charges-icon" />
                    <span>Fees and Charges</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={Transactions} alt="transactions-icon" />
                    <span>Transactions</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={Services} alt="services-icon" />
                    <span>Services</span>
                </Link>
                <Link to='/'>
                    <img src={ServiceAmount} alt="service-amount-icon" />
                    <span>Service Account</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={SettleMents} alt="settlements-icon" />
                    <span>Settlements</span>
                </Link>
                <Link onClick={hanldeSetShowSideBar} to='/'>
                    <img src={Reports} alt="reports-icon" />
                    <span>Reports</span>
                </Link>

                {/* end section */}
                <span className='end-section'>
                    <h3>Businesses</h3>
                    <Link onClick={hanldeSetShowSideBar} to='/'>
                        <img src={Preferences} alt="preferences-icon" />
                        <span>Preferences</span>
                    </Link>
                    <Link onClick={hanldeSetShowSideBar} to='/'>
                        <img src={Pricing} alt="fees-and-pricing-icon" />
                        <span>Fees and Pricing</span>
                    </Link>
                    <Link onClick={hanldeSetShowSideBar} to='/'>
                        <img src={AuditLogs} alt="audit-logs-icon" />
                        <span>Audit Logs</span>
                    </Link>
                    <Link onClick={hanldeSetShowSideBar} to='/'>
                        <img src={SystemMessages} alt="systen-messages-icon" />
                        <span>System Messages</span>
                    </Link>
                </span>


                <div className="log-out-section">
                    <span onClick={()=>navigate('/login')} className='log-out'>
                        <img src={Logout} alt="log-out-icon" />
                        <span>Logout</span>
                    </span>

                    <span className='v1'>
                        v1.2.0
                    </span>
                </div>



            </section>
            <section className="bottom"></section>
        </div>
    )
}
