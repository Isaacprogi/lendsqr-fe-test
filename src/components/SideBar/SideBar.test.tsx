import { render } from '@testing-library/react';
import { SideBar } from './SideBar'
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer'



describe('test for SideBar component', () => {
  test('test for SideBar component', () => {
    render(<BrowserRouter><SideBar /></BrowserRouter>)
  })

  test('should match snapshot', async () => {
    const tree = renderer.create(<BrowserRouter><SideBar /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
})























