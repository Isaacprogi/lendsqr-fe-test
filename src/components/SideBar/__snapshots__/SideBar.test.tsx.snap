// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test for SideBar component > should match snapshot 1`] = `
<div
  className="sidebar"
>
  <section
    className="profile-more-options"
    data-testid="profile-more-options"
  >
    <div
      className="profile-info"
    >
      <span
        className="avatar"
      >
        <img
          alt=""
          data-testid="avatar"
          src="/src/assets/images/Profile.png"
        />
      </span>
      <span
        data-testid="user-name-2"
      >
        Adeji
      </span>
    </div>
    <a
      href="/"
      onClick={[Function]}
    >
      <div
        className="top-bottom-icon-duo"
      >
        <svg
          data-testid="notification-drawer-icon"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            {
              "color": undefined,
            }
          }
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M816 768h-24V428c0-141.1-104.3-257.8-240-277.2V112c0-22.1-17.9-40-40-40s-40 17.9-40 40v38.8C336.3 170.2 232 286.9 232 428v340h-24c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h216c0 61.8 50.2 112 112 112s112-50.2 112-112h216c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM512 888c-26.5 0-48-21.5-48-48h96c0 26.5-21.5 48-48 48z"
          />
        </svg>
      </div>
      <span>
         Notifications
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <div
        className="top-bottom-icon-duo"
      >
        <svg
          aria-hidden="true"
          data-testid="docs-drawer-icon"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            {
              "color": undefined,
            }
          }
          viewBox="0 0 20 20"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clipRule="evenodd"
            d="M4 4a2 2 0 012-2h4.586A2 2 0 0112 2.586L15.414 6A2 2 0 0116 7.414V16a2 2 0 01-2 2H6a2 2 0 01-2-2V4z"
            fillRule="evenodd"
          />
        </svg>
      </div>
      <span>
        Docs
      </span>
    </a>
  </section>
  <section
    className="top"
  >
    <a
      className="org"
      href="/"
      onClick={[Function]}
    >
      <div
        className="top-bottom-icon-duo"
      >
        <img
          alt="switch-organization-icon-1"
          src="/src/assets/svg/sidebar/OrganizationTop.svg"
        />
        <img
          alt="switch-organization-icon-2"
          src="/src/assets/svg/sidebar/OrganizationBottom.svg"
        />
      </div>
      <span>
        Switch Organization
      </span>
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          {
            "color": undefined,
          }
        }
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M16.293 9.293 12 13.586 7.707 9.293l-1.414 1.414L12 16.414l5.707-5.707z"
        />
      </svg>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <div
        className="top-bottom-icon-duo"
      >
        <img
          alt="dashboard-icons-1"
          src="/src/assets/svg/sidebar/DashBoardTop.svg"
        />
        <img
          alt="dashboard-icon-2"
          className="dash-board-bottom"
          src="/src/assets/svg/sidebar/DashBoardBottom.svg"
        />
      </div>
      <span>
        Dash Board
      </span>
    </a>
  </section>
  <section
    className="middle"
  >
    <h3>
      Customers
    </h3>
    <a
      href="/users"
      onClick={[Function]}
    >
      <img
        alt="users-icon"
        src="/src/assets/svg/sidebar/Users.svg"
      />
      <span>
        Users
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="guarantors-icon"
        src="/src/assets/svg/sidebar/Guarantors.svg"
      />
      <span>
        Guarantors
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="loans-icon"
        src="/src/assets/svg/sidebar/Loans.svg"
      />
      <span>
        Loans
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="decision-icon"
        src="/src/assets/svg/sidebar/Decision.svg"
      />
      <span>
        Decision Model
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="savings-icon"
        src="/src/assets/svg/sidebar/Savings.svg"
      />
      <span>
        Savings
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="loan-request-icon"
        src="/src/assets/svg/sidebar/LoanRequests.svg"
      />
      <span>
        Loan Requests
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="white-list-icon"
        src="/src/assets/svg/sidebar/WhiteList.svg"
      />
      <span>
        whiteList
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="karma-icon"
        src="/src/assets/svg/sidebar/Karma.svg"
      />
      <span>
        Karma
      </span>
    </a>
    <h3>
      Businesses
    </h3>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="organization-icon"
        src="/src/assets/svg/sidebar/briefcase.svg"
      />
      <span>
        Organization
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="loan-products-icon"
        src="/src/assets/svg/sidebar/Guarantors.svg"
      />
      <span>
        Loan Products
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="savings-product-icon"
        src="/src/assets/svg/sidebar/SavingsNew.svg"
      />
      <span>
        Savings Products
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="fees-and-charges-icon"
        src="/src/assets/svg/sidebar/fees.svg"
      />
      <span>
        Fees and Charges
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="transactions-icon"
        src="/src/assets/svg/sidebar/trans.svg"
      />
      <span>
        Transactions
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="services-icon"
        src="/src/assets/svg/sidebar/Services.svg"
      />
      <span>
        Services
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="service-amount-icon"
        src="/src/assets/svg/sidebar/ServiceAmount.svg"
      />
      <span>
        Service Account
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="settlements-icon"
        src="/src/assets/svg/sidebar/settlements.svg"
      />
      <span>
        Settlements
      </span>
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="reports-icon"
        src="/src/assets/svg/sidebar/chart-bar.svg"
      />
      <span>
        Reports
      </span>
    </a>
    <span
      className="end-section"
    >
      <h3>
        Businesses
      </h3>
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="preferences-icon"
          src="/src/assets/svg/sidebar/Preferences.svg"
        />
        <span>
          Preferences
        </span>
      </a>
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="fees-and-pricing-icon"
          src="/src/assets/svg/sidebar/pricing.svg"
        />
        <span>
          Fees and Pricing
        </span>
      </a>
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="audit-logs-icon"
          src="/src/assets/svg/sidebar/auditLogs.svg"
        />
        <span>
          Audit Logs
        </span>
      </a>
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="systen-messages-icon"
          src="/src/assets/svg/sidebar/sysmessages.svg"
        />
        <span>
          System Messages
        </span>
      </a>
    </span>
    <div
      className="log-out-section"
    >
      <span
        className="log-out"
        onClick={[Function]}
      >
        <img
          alt="log-out-icon"
          src="/src/assets/svg/sidebar/sign-out.svg"
        />
        <span>
          Logout
        </span>
      </span>
      <span
        className="v1"
      >
        v1.2.0
      </span>
    </div>
  </section>
  <section
    className="bottom"
  />
</div>
`;
